kuzzleio/kuzzle
# merge seoUpgrade into master

## Upgrade seo for documentation 


### How should this be manually tested?

  - Step 1 : merge and build
  - Step 2 : check the metas on the prod doc 
  - Step 3 : ask google for a reindexation 

ci-allow-merge-into: master
